jobs/build/scan-images/Jenkinsfile

```diff
@@ -17,18 +17,26 @@ properties([[
 
 node('openshift-build-1') {
   try {
-    stage('clone') { dir('aos-cd-jobs') { checkout scm } }
+    stage('clone') {
+      dir('aos-cd-jobs') { checkout scm }
+      dir('oit') { git 'https://github.com/openshift/enterprise-images.git' }
+    }
+    stage('venv') {
+      if(!fileExists('env')) { sh 'virtualenv env' }
```

Comment:
Any environment variables populated for the virtual environment here will be lost as soon as sh returns here. This is the pattern used in other Jenkins pipelines: https://github.com/adammhaile/aos-cd-jobs/commit/60eab300ea85d2582c1c540cab175f34e51e1454#diff-4afa940aacab03ebeba0c98c8cc67d64R194

Reply:
This command only creates the virtualenv the first time the job is executed.
All others should fail the `!fileExists('env')` test.  The virtualenv is used
on line 35, where `env/bin/python` is used instead of `python` or
`/bin/python`.

That has the same effect (again, it worked on all my tests), but I don't object
to a standard way of using virtualenvs.  Maybe something for the common libs?
I'll make the change to the job anyway.